src/main/webapp/index.html

```diff
@@ -22,5 +22,22 @@ <h2>COMPSCI:GO</h2>
     </div>
   </div>
 
+  <!--small description-->
+  <div class="step-wrap, step-align">
+    <p>
+      The objective of this capstone project is to create a web application
```

Comment:
Can we provide links to data?

It would be awesome if the user could be redirected to other places to learn about some of the disparities.

Reply:
Great Idea !!!

Reply:
These line breaks seem inconsistent. Is there a way to add the linting for css and html?